task/control-plane-agent/src/mgs_common.rs

```diff
@@ -192,20 +191,21 @@ impl MgsCommon {
                         //     (Management plane should do that.)
                         //   - Enable staged updates where we don't automatically
                         //     reset after writing an image.
-                        ringbuf_entry!(Log::RotReset {
-                            err: SprotError::RspTimeout
-                        });
+                        ringbuf_entry!(Log::RotReset);
                         Ok(())
                     }
-                    Err(err) => {
+                    Err(_) => {
                         // Some other error occurred.
                         // Update is all-or-nothing at the moment.
                         // The control plane can try to reset the RoT again or it
                         // can start the update process all over again.  We should
                         // be able to make incremental progress if there is some
                         // bug/condition that is degrading SpRot communications.
-                        ringbuf_entry!(Log::RotReset { err });
-                        Err(SpError::ComponentOperationFailed(err.into()))
+
+                        // TODO(AJS): We need to be able to expose all SprotErrors directly
+                        // and not just as u32s
+                        //ringbuf_entry!(Log::RotReset { err });
+                        Err(SpError::ComponentOperationFailed(9999))
```

Comment:
nein nein nein nein ;-)

Reply:
laughed out loud when I read this last night.